### PR TITLE
services/horizon/ingest: Close LedgerBackend in Shutdown

### PR DESCRIPTION
**Pull request by bartekn**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

TODO check if it works correctly with https://github.com/stellar/go/pull/3258.

### Why

[TODO: Why this change is being made. Include any context required to understand the why.]

### Known limitations

[TODO or N/A]

### CODE REVIEW

File: services/horizon/internal/app.go [hunk 1]
```diff
@@ -158,7 +158,10 @@ func (a *App) waitForDone() {
 	a.webServer.Shutdown(webShutdownCtx)
 	a.cancel()
 	if a.ingester != nil {
-		a.ingester.Shutdown()
+		err := a.ingester.Shutdown()
+		if err != nil {
```
Comment (2opremio) [inline services/horizon/internal/app.go:162]: You can inline the shutdown call here

**Comment by 2opremio** [comment]
It would be cool to add a test which actually covers signal management.

That may require mocking the signal notification.

**Review by 2opremio — APPROVED** [review]
Minor nitpick and (optional) suggestion of testing signal handling.

**Comment by tamirms** [comment]
@bartekn I think you can close this PR because the same functionality was implemented in https://github.com/stellar/go/pull/3278/files#diff-933b9cab99da778822646614e836dc6c6b0cae788d582b977e896c86a67bd5d8R516

**Comment by bartekn** [comment]
Thanks @tamirms!

**Pull request closed without merging**